src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -318,15 +364,15 @@ Astro will log a **warning** to the terminal:
 [@matthewp/my-adapter] The feature is experimental and subject to issues or changes.
```

Comment:
This seems like unnecessarily breaking up mid sentence here, what about something like: 

If the Sharp image service is used, Astro will log a warning and error to the terminal based on your adapter's support:

```
[@matthewp/my-adapter] The feature is experimental and subject to issues or changes.

[@matthewp/my-adapter] The currently selected adapter `@matthewp/my-adapter` is not compatible with the service "Sharp". Your project will NOT be able to build.
```